boot/bootutil/src/bootutil_priv.h

```diff
@@ -406,10 +406,14 @@ boot_img_sector_off(const struct boot_loader_state *state, size_t slot,
 #endif  /* !defined(MCUBOOT_USE_FLASH_AREA_GET_SECTORS) */
 
 #ifdef MCUBOOT_RAM_LOAD
+#define IMAGE_RAM_BASE ((uintptr_t)3)
```

Comment:
Might not have got it fully, but this seems to have an impact not only on the simulator environment rather the real targets as well.

Reply:
Look like I see the same. ^^

Reply:
Yeah, this isn't done yet, just a placeholder to get it to compile in the simulator. I'm working on actually testing these features now, which will involve putting something meaningful here.

Reply:
Could you set the macro value to 0? Then it won't break the functionality and its value is changed anyway to 0 in a later commit.